contrib/asff.tpl

```diff
@@ -13,15 +17,15 @@
     {{- if eq $severity "UNKNOWN" -}}
     {{- $severity = "INFORMATIONAL" -}}
     {{- end -}}
-    {{- $description := .Description -}}
+    {{- $description := escapeString .Description | printf "%q" -}}
     {{- if gt (len $description ) 1021 -}}
         {{- $description = (substr 0 1021 $description) | printf "%v .." -}}
     {{- end}}
         {
             "SchemaVersion": "2018-10-08",
             "Id": "{{ $target }}/{{ .VulnerabilityID }}",
             "ProductArn": "arn:aws:securityhub:{{ env "AWS_REGION" }}::product/aquasecurity/aquasecurity",
-            "GeneratorId": "Trivy",
+            "GeneratorId": "Trivy/{{ .VulnerabilityID }}",
```

Comment:
I'm wondering about `GeneratorId`. I took a loot at the documentation and found next:
> GeneratorId
> The identifier for the solution-specific component (a discrete unit of logic) that generated a finding. In various solutions from security findings products, this generator can be called a rule, a check, a detector, a plugin, and so on.

But the common way to use `GeneratorId` is for grouping vulnerabilities, isn't it?

Reply:
That's right, @afdesk . Some security format conversion tools (e.g. [SAF](https://saf-cli.mitre.org/)) group Trivy findings incorrectly because they all currently share the same static `GeneratorId` field of 'Trivy'.
Most other security scan tools, including [kube-bench](https://github.com/aquasecurity/kube-bench/blob/be157a8a5ad9f931a107ab39db2ab30b7f4cbd00/check/controls.go#L251), append a control/vulnerability ID to the `GeneratorId` field so that like findings can be grouped together.

Reply:
@ben-harvey thanks for the explanation! it seems that it makes sense